tests/dataimportcron_test.go

```diff
@@ -691,6 +692,12 @@ var _ = Describe("DataImportCron", func() {
 			err = f.CrClient.List(context.TODO(), snapshots, &client.ListOptions{Namespace: ns})
 			Expect(err).ToNot(HaveOccurred())
 			Expect(snapshots.Items).To(BeEmpty())
+			// Ensure existing PVC clones from this source don't mess up future ones
+			cloneDV := utils.NewDataVolumeForImageCloningAndStorageSpec("target-dv-from-pvc", size, ns, currentImportDv, nil, nil)
+			cloneDV, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, cloneDV)
+			Expect(err).ToNot(HaveOccurred())
+			f.ForceBindPvcIfDvIsWaitForFirstConsumer(cloneDV)
+			err = utils.WaitForDataVolumePhase(f, cloneDV.Namespace, cdiv1.Succeeded, cloneDV.Name)
```

Comment:
Not sure I understand the need for this check

Reply:
This is the use case that triggers this nil ptr;

A cloned DV from a previous source PVC named `foo` exists
You upgrade CDI, now the dataimprotcron controller has a snapshot source `foo` created
This causes a DV with source.pvc to be queued in the snapshot clone controller